tests/images/build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM oryxdevms/build:latest
+FROM oryxdevmcr.azurecr.io/public/oryx/build:latest
```

Comment:
Would this change cause a problem locally? like with this repo name exist? This matters locally because our tests use `oryxtests/build` image to run tests so we should be able to build this image successfully locally too.

Reply:
I will revert this change as I'm tagging oryxdevms/build